backend/src/impact.rs

```diff
@@ -9,26 +9,22 @@ use crate::{od, MapModel, RoadID};
 /// see how traffic changes across roads in the whole map. This works by finding the best route
 /// before and after changes for every origin/destination "OD" pairs, then counting routes per
 /// road.
+#[derive(Default)]
 pub struct Impact {
     // (r1, r2, count) -- `count` identical trips from `r1` to `r2`
-    requests: Vec<(RoadID, RoadID, usize)>,
+    sampled_requests: Vec<(RoadID, RoadID, usize)>,
+    all_requests: Vec<(RoadID, RoadID, usize)>,
```

Comment:
nit: the usize count is redundant here, it's always 1. If it could help memory, we could `iter().map()` to make the two look the same.

Reply:
I'm not very familiar with this part of the app yet, but plausibly there are duplicated entries in `all_requests`? Maybe we could leverage this usize to dedupe them.

No need to do that in this PR, but it might save more space than just getting rid of the usize.

Reply:
#129 has this idea, thanks! I'm hoping the benchmark started will capture the effects of incremental improvements like this